elk/extraction/extraction.py

```diff
@@ -9,7 +9,7 @@
 import torch.distributed as dist
 
 
-@torch.autocast("cuda", enabled=torch.cuda.is_available())
+@torch.autocast("cuda", enabled=torch.cuda.is_available())  # type: ignore
```

Comment:
I'm not seeing typing issues here. Are you using mypy?

Reply:
I think this was a pylance error. I'll check

Reply:
Pylance
Pytorch 1.13.1+cu117

Reply:
I don't see any issues here